x/ccv/provider/keeper/keeper.go

```diff
@@ -761,12 +759,19 @@ func (k Keeper) GetSlashAcks(ctx sdk.Context, chainID string) []string {
 }
 
 // ConsumeSlashAcks empties and returns the slash acks for a given chain ID
-func (k Keeper) ConsumeSlashAcks(ctx sdk.Context, chainID string) (acks []string) {
-	acks = k.GetSlashAcks(ctx, chainID)
-	if len(acks) < 1 {
+func (k Keeper) ConsumeSlashAcks(ctx sdk.Context, chainID string) (acks []types.ConsumerConsAddress) {
```

Comment:
ConsumeSlashAcks is only used in one place in `relay.go` which accepts `[]string` anyways. Seems reasonable to return strings here to reduce complexity